README.md

```diff
@@ -472,6 +487,185 @@ async function saveCoverageData (codeBlocks) {
 export default runUnitTests
 ```
 
+
+## running linter
+```js
+// runLinter.js
+import fs from 'fs'
+import saveToFile from './saveToFile'
+import standard from 'standard'
+import forEachCodeBlock from './forEachCodeBlock'
+
+export const paddingText = (width, string, padding = ' ') => {
+  return (width <= string.length) ? string : paddingText(width, string + padding, padding)
+}
+
+export const countLinterErrors = (results) => results[0].messages.length
+
+export const removeEslintDisable = (remover) => (result) => {
+  if (result.output) {
+    result.output = result.output.replace(remover, '')
+  }
+  return result
+}
+
+export const runStandardLinter = (contents, fix) => (
+  new Promise((resolve, reject) => {
+    const disabledRules = ['no-undef', 'no-unused-vars', 'no-lone-blocks', 'no-labels']
+    const eslintDisable = '/* eslint-disable ' + disabledRules.join(', ') + ' */\n'
+    standard.lintText(eslintDisable + contents, { fix }, (err, { results }) => {
```

Comment:
Would it be better to use ESLint’s API with `eslint-config-standard` instead? This allows us to override any variable.

Also, I am not sure why some of these rules have to be disabled. Each code block inside an essay usually corresponds to a single ES module, so there should be no unused variables? Would be great if there is an example for these exceptions (if there should be exceptions).